src/dndbeyond/content-scripts/character.js

```diff
@@ -277,9 +294,84 @@ function rollAbilityOrSavingThrow(paneClass, rollType) {
     if (character.hasClassFeature("Trance of Order") && character.getSetting("sorcerer-trance-of-order", false))
             roll_properties.d20 = "1d20min10";
 
+    // effects 
+    if(rollType == "saving-throw" && character.getSetting("effects-bless", false)) {
+        roll_properties["modifier"] += "+1d4";
+        addEffect(roll_properties, "Bless");
+    } else if(rollType == "saving-throw" && character.getSetting("effects-bane", false)) {
+        roll_properties["modifier"] += "-1d4";
+        addEffect(roll_properties, "Bane");
+    }
```

Comment:
I think doing the bless/bane in one place is better. Here you're doing it for ability and saving throws, but then the check happens again in `sendRoll` but only checks for attack and spell-attack.
It's better to remove it from here and expand the list of roll types the check in `sendRoll` applies it to.